vhdbuilder/packer/install-dependencies.sh

```diff
@@ -321,27 +321,43 @@ for OMS_AGENT_IMAGE in ${OMS_AGENT_IMAGES}; do
 done
 
 # calico images used by AKS
-CALICO_CNI_IMAGES="v3.5.0"
+CALICO_CNI_IMAGES="
+v3.5.0
+v3.8.0
```

Comment:
Calico is bumped to v3.8.0 in AKS https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp/pullrequest/3006037